usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -14,11 +14,18 @@
 import time
 import traceback
 
-from datetime import datetime, timezone
-from ddtrace import tracer
-from ddtrace.ext import SpanTypes
+import django
 from django.conf import settings
 
+from opentelemetry import trace
+from opentelemetry.trace import SpanKind
+
+from datetime import datetime, timezone
+from django.apps import apps
+
+if not apps.ready:
```

Comment:
Flagging this to discuss later. Not a big fan of adding this here especially without understanding specifically why we need to add it to begin with. While it works, I would say we can use it more to figure out what the actual problem is instead of as a solution.

Reply:
Took this out in the latest change when I pinned the otel package versions. lets see if this is no longer needed. Hopefully not.

Reply:
YAY took it out and tests passed!